north_extensions

```diff
@@ -0,0 +1,24 @@
+C0.gbr.fitacf
```

Comment:
This 'C0' is our own (non-standardized) syntax for 24-hour concatenated files. Should make this more portable by not requiring the 'C0' and also, does this script require 24-hour files?